docs/Segments.md

```diff
@@ -227,39 +239,34 @@ Links the `.bss` section of the associated `c` file.
 - { start: 0x7D1AD0, type: .bss, name: filepath, vram: 0x803C0420 }
 ```
 
-## Images
+## `.sbss`
 
-**Description:**
+The `.sbss` segment behaves the same as the `.bss` segment but supports "small bss" linker sections named `.sbss`.
 
-**splat** supports most of the [N64 image formats](https://n64squid.com/homebrew/n64-sdk/textures/image-formats/):
-
-- `i`, i.e. `i4` and `i8`
-- `ia`, i.e. `ia4`, `ia8`, and `ia16`
-- `ci`, i.e. `ci4` and `ci8`
-- `rgb`, i.e. `rgba32` and `rgba16`
+## `lib`
 
-These segments will parse the image data and dump out a `png` file.
+The `lib` segment can be used to link to a section of an object in an existing library. It is purely used to configure the output linker script and does not do any extraction.
 
-**Note:** Using the dictionary syntax allows for richer configuration.
+It looks for libraries in the [`lib_path`](https://github.com/ethteck/splat/wiki/Configuration#lib_path) global option.
 
 **Example:**
 
+(link to .text of b_obj in a_lib)
```

Comment:
Could move all these (brackets) to `#comments` inside each example like other sections?